tools/create_project.py

```diff
@@ -0,0 +1,26 @@
+def create_project(name):
+    if exists(name):
+        print('Project %s already exists' % name)
+        return
+    print('Creating project %s' % name)
+    new_project = Project.objects.create(linkname=name,
+            name=name,
+            listid=name,
+            listemail='{0}@{0}.com'.format(name))
+
+def exists(project_name):
+    return len(Project.objects.filter(linkname=project_name)) > 0
+
+def find_project_id(name):
+    project = Project.objects.get(linkname=name)
+    return project.id
+
+
+if __name__ == '__main__':
+    import django
+    django.setup()
+    from patchwork.models import Project
+    import sys
+    create_project(sys.argv[1])
+    with open('project_id', 'w+') as f:
+        f.write(str(find_project_id(sys.argv[1])))
```

Comment:
BTW: What I just thought of: Why can't we use cURL to simply do API calls to patchwork to create the project? Just a thought...

Reply:
And: This file is not related to PaStA, and it's not a pasta tool. Please push that file to a fork of patchwork. The integration shall then checkout patchwork and switch to the appropriate branch.

Reply:
> BTW: What I just thought of: Why can't we use cURL to simply do API calls to patchwork to create the project? Just a thought...

There is no API in Patchwork to create projects as of now

Reply:
Sure, there is. Otherwise you wouldn't be able to create a project with the web interface. You know the request that will be submitted when you create a project. That can also be called via curl.

Reply:
I mean there is no publically exposed API for that. That probably uses some internal API. Reference: https://patchwork.readthedocs.io/en/latest/api/rest/schemas/v1.2/

Reply:
POST request on http://localhost:8000/admin/patchwork/project/add/, linkname: foo, listemail: a@b.com, ...

Reply:
Would be a single line in the shell script. Though I don't know how complex it is to forward cookies, ... So that's a thing with really low priority.

Reply:
Hmm yeah. Anyway doing it through python is reliable enough, because I am assuming if we use the admin API we will have to create an admin account too. And then somehow get a cookie to send along with the request

Reply:
Yes true, so this is really low prio.